saas_app/initialize.sh

```diff
@@ -40,3 +40,26 @@ if [ "${users_length}" -eq 0 ]; then
 else
     echo "There are users"
 fi  
+
+# Load the fixtures data so that the database is populated with some data for testing purposes
+# check to see if Soial account is installed
+social_account=$(python manage.py shell -c "from allauth.socialaccount.models import SocialApp; print(len(SocialApp.objects.all()))")
+if [ "${social_account}" -eq 0 ]; then
```

Comment:
Using: Codespaces in VS Code (not the browser)

Something doesn't seem to properly get set up for me, even with Codespaces. I rebuilt the Codespaces from scratch which ran the initialize script and it doesn't have the social app setup which gives me this error:
<img width="754" alt="image" src="https://user-images.githubusercontent.com/1690085/222764233-80d0d6c9-486c-4937-9242-a42ef73795b5.png">